Sources/Peripheral/Peripheral.swift

```diff
@@ -55,14 +55,14 @@ public class Peripheral {
         #endif
     }
     
-    let cbPeripheral: CBPeripheral
+    public let cbPeripheral: CBPeripheral
     
     private let context: PeripheralContext
     /// The delegate object that will receive `cbPeripheral`.
     /// - Note: We need to hold on to it because `cbPeripheral` has a weak reference to it.
     private let cbPeripheralDelegate: DelegateWrapper
     
-    init(_ cbPeripheral: CBPeripheral) {
+    public init(_ cbPeripheral: CBPeripheral) {
```

Comment:
I worry that exposing this constructor will encourage the use of CoreBluetooth types directly. That can lead to errors since users will get a chance to change the peripheral's delegate, which will break this lib's async/await strategy.